en-US/win10/SetupBoardManually.md

```diff
@@ -0,0 +1,85 @@
+<div class="row">
+  <div class="col-md-6 col-sm-12 col-no-padding">
+    <p>The set up experience for the latest insider image is slightly different than our other products. The following flow is to help you get going with our most recent builds (hot off the press!).</p>
+  </div>
+</div>
+
+<ol class="inline-list">
+  <div class="row">
+    <div class="col-md-6 col-sm-12 col-no-padding"> 
+      <li><b>Visit the downloads page <a href="http://go.microsoft.com/fwlink/p/?linkID=532967" target="_blank">here</a></b>.</li>
+    </div>
+    <div class="col-md-6 col-sm-12">
+
+    </div>
+   </div>
+   
+   <div class="row">
+    <div class="col-md-6 col-sm-12 col-no-padding">
+      <li><b>Click the desired image under the “Windows 10 IoT Core Insider Preview Image”.</b> You are required to have a Microsoft account and must a part of the Windows Insider Program to gain access to this image. Joining the Windows Insider Program is free and anyone can join.
+      Once signed up, your image will automatically begin to download. </li>
+    </div>
+    <div class="col-md-6 col-sm-12">
+
+    </div>
+   </div>
+
+   <div class="row">
+    <div class="col-md-6 col-sm-12 col-no-padding">
+      <li><b>Find the downloaded .ISO and double click on it.</b></li>
+    </div>
+    <div class="col-md-6 col-sm-12">
+      <img src="{{site.baseurl}}/images/ManualSetup/Manualstep1.png">
+    </div>
+   </div>
+   
+   <div class="row">
+    <div class="col-md-6 col-sm-12 col-no-padding">   
+      <li><b>Double click the .MSI and follow the installation steps</b></li>
+    </div>
+    <div class="col-md-6 col-sm-12">
+      <img src="{{site.baseurl}}/images/ManualSetup/Manualstep2.png">
+      <img src="{{site.baseurl}}/images/ManualSetup/Manualstep3.png">
+    </div>
+   </div>
+   
+   <div class="row">
+    <div class="col-md-6 col-sm-12 col-no-padding">   
+      <li><b>Launch the IoT Dashboard.</b> The Windows 10 IoT Core dashboard will allow you to flash the downloaded image onto your Sd card. You can download the IoT Dashboard <a href="http://go.microsoft.com/fwlink/?LinkID=708576" target="_blank">here.</a> </li>     
+    </div>
+    <div class="col-md-6 col-sm-12">
+    </div>
+   </div>    
+      
+   <div class="row">
+    <div class="col-md-6 col-sm-12 col-no-padding">         
+      <li><b>Click ”Set up a new device”.</b> Once there, select “Custom” from the "Device type" drop down. Click the "Browse" button to find your .ffu </li>
```

Comment:
Not sure if this is the correct way to say this or not (it absolutely could be), but the '.' in "to find your .ffu" seems odd to me.